Ruihao zhang (#21)
hcts-hra/heidenc: content/bibliography.md

Shen Tongsheng 沈桐生, Guiji Shen shi Dong Xi xueshu tiyao zongxu 會稽沈氏東西學書提要總敘,2 juan.Shanghai: Duyouyongshuzhai, 1897. Preface by the author written in Shanghai.

Availability: Open access at [Hathi Trust Digital Library](https://hdl.handle.net/2027/uc1.b3918054); Scan in HD database.

---

*Xishi lice* 西事蠡測, Shen Chun 沈純, Short selection of random pieces that have to do with West. No order visible. No sources given. Some explanatory matter in small print added. Contains info up to 1882, probably published 1883.

Printed in Wang Xiqi 王錫祺(1855-1913) comp., Xiaofanghu zhai yudi congchao 小方壺齋輿地叢鈔 vol.15, 18 pp. Shanghai Shuyi tang \[1879\] , reprint Hangzhou 1985.

Availability: 小方壺齋輿地叢鈔 can be accessed at [[Hathi Trust Digital Library]](https://catalog.hathitrust.org/api/volumes/oclc/502895999.html), in which 西事蠡測 can be found in vol.62; SCAN in Heidelberg database.

---

*Wuzhou fangyu kao* 五洲方域考, Gong Chai 龔柴, ed., in Wang Xiji 王錫輯 (1855-1913) comp., *Xiaofanghuzhai yudi congchao* (1879). Date of original not clear

> Availability: 小方壺齋輿地叢鈔 can be accessed at [[Hathi Trust Digital Library]](https://catalog.hathitrust.org/api/volumes/oclc/502895999.html), in which 五洲方域考 in vol.1; Print copy Heidelberg

Reviewer: ```suggestion
Availability: 小方壺齋輿地叢鈔 can be accessed at [Hathi Trust Digital Library](https://catalog.hathitrust.org/api/volumes/oclc/502895999.html), in which 五洲方域考 in vol.1; Print copy Heidelberg
```

Availability: copy of 1898 edition in Heidelberg, SCAN in database under *Jingji tongkao*.

> 3rd ed. 1901. Harvard has the same copy, cover title *Jingji tongkao*. Ying Zuxi應祖錫and Han Qingfu 韓卿甫, zuan 纂。

Reviewer: ```suggestion
3rd ed. 1901. Harvard has the same copy, cover title *Jingji tongkao*. Ying Zuxi 應祖錫 and Han Qingfu 韓卿甫, zuan 纂。
```

Zhongyang yanjiuyuan, jinshisuo, nangang. Academia Sinica.

Availability: It can be downloaded for free at [Wikimedias Commons](https://commons.wikimedia.org/wiki/File:CADAL01036937_%E8%90%AC%E5%9C%8B%E6%94%BF%E6%B2%BB%E8%90%9F%E5%AD%B8%E5%85%A8%E6%9B%B8%C2%B7%E8%90%AC%E5%9C%8B%E8%97%9D%E5%AD%B8%E5%8F%A2%E8%80%83.djvu);Scan from Academia Sinica included in Heidelberg database.

---

*Xixue san tong* 《西學三通》. Zhang Jian 張謇 (1853-1926) writes character on cover. Xie Ruochao 謝若潮 jian ding 鑒定, conceptualized, Yuan Zonglian 袁宗濂 Yan Zhiqing 晏志清 bian ji, compiled. Preface Guangxu 28 (1902) by Xie Ruochao. , Shanghai 萃新書館藏珍 , printed by Shanghai 文盛堂 with  lithography. 

Section 1: \[v. 1-40\] 西政通典 Xizheng tongdian 
Section 2: \[v. 41-72\] 西史通志 Xi shi tongzhi 
Section 3: \[v. 73-130\] 西藝通考. Xi yi tong kao. Preface in section 1: Tao Jun 陶鈞。Same in section 2 and 3. Advertisement on this book in Zhongwai ribao Jan 1, 1902, text translated by I. Amelung, "Encyclopedic publications", Conference paper, Heidelberg April 2016.

Availability: SCAN in Heidelberg database.
Other copies: Australian National University.

---

> *Yixue tongzuan* 《藝學通纂》, (*Compendium of Technical Learning)*：14 類 88卷12冊。Ma Jianzhong ed., 馬建忠 。光緒壬寅年（1902 ）。Preface by Luo Zhenyu 羅振玉Shanghai, Shanghai Wenlin, lithograph, 上海文林石印。Summary in paper by Li Xiaoti, Jianli xi shiye, pp. 3-7.

Reviewer: ```suggestion
*Yixue tongzuan* 《藝學通纂》, (*Compendium of Technical Learning)*：14 類 88卷12冊。Ma Jianzhong ed., 馬建忠 。光緒壬寅年（1902 ）。Preface by Luo Zhenyu 羅振玉 Shanghai, Shanghai Wenlin, lithograph, 上海文林石印。Summary in paper by Li Xiaoti, Jianli xi shiye, pp. 3-7.
```
